services/worker-manager/test/fake-azure.js

```diff
@@ -77,45 +87,43 @@ class FakeAzure {
           },
         ],
       },
-      vmId: "123",
+      vmId: "a-vm-id",
       provisioningState: 'Creating',
     };
 
     const azureError = new Error("something went wrong");
 
-    const getVMStub = sinon.stub();
-    // first call returns provisioningState Creating, second returns Deallocated
-    getVMStub.onCall(0).returns(instanceData);
-    getVMStub.onCall(1).returns({...instanceData, provisioningState: 'Deallocated'});
+    this.getVMStub = sinon.stub();
+    // first call returns provisioningState Creating, second returns Failed
+    this.getVMStub.onCall(0).returns({...instanceData, provisioningState: 'Succeeded'});
```

Comment:
Comment says "Creating" but the code says "Succeeded"??

Reply:
Fixed the comment. Essentially, I hadn't differentiated between `provisioningStates` and `powerStates`, and now we are.